modules.json

```diff
@@ -5,5 +5,6 @@
   "trustxBidAdapter",
   "improvedigitalBidAdapter",
   "appnexusBidAdapter",
-  "guAnalyticsAdapter"
+  "guAnalyticsAdapter",
+  "prebidServerBidAdapter"
```

Comment:
Perhaps keep all the bid adapters before the analytics adapter?

Reply:
Or maybe better to put the analytics adapter before all the bid adapters because there will be more bid adapters coming.